### PR TITLE
React testing

### PR DESCRIPTION
**Pull request by d3mola**
What does this PR do?
Tests actions and reducers of the client side code base and clean up code

Description of Task to be completed?
Install jest
Writ unit tests fro actions and reducers
Clean up unused code

How should this be manually tested?
run `npm run test:client`

Follow readme instructions to start client and server
On POSTMAN navigate to DELETElocalhost:8080/events/:id
Any background context you want to provide?
Only an admin can access this route
What are the relevant pivotal tracker stories?

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
-  "presets": ["react", "env"],
+  "presets": ["react", "env", "stage-2"],
   "plugins": [
-    "transform-object-rest-spread"
+    "transform-object-rest-spread",
+    "transform-class-properties"
   ]
 }
```
Comment (tomily1) [inline .babelrc:7]: nitpick: This red icon indicates a new line should be below it. you might want to note that in the future

File: client/Sagas/asyncOperations.js [hunk 2]
```diff
@@ -13,350 +13,345 @@ axios.defaults.headers.common['x-access-token'] = token;
...
+//  * Aysnc operation to sign up
+//  */
+
+// /**
+//  * worker saga performs async calls to the api
+//  * @export
+//  * @param {any} action
+//  * @returns {object} response
+//  */
+// export function* signUpAsync(action) {
```
Comment (tomily1) [inline client/Sagas/asyncOperations.js:26]: remove commented codes since they are not useful

File: client/Sagas/asyncOperations.js [hunk 3]
```diff
@@ -366,42 +361,39 @@ export function* watchshowEditFormAsync() {
...
+// export function* watchDeleteCenterAsync() {
+//   yield takeEvery(actionTypes.DELETE_CENTER, deleteCenterAsync);
+// }
+
+// export function* editCenterAsync(action) {
+//   try {
+//     const response = yield call(axios.put, `${localUrl}/api/v1/centers/${action.center.id}`, {
+//       token,
+//       name: action.center.name,
+//       capacity: action.center.capacity,
```
Comment (tomily1) [inline client/Sagas/asyncOperations.js:383]: same her, commented codes

File: client/store.js [hunk 4]
```diff
@@ -1,20 +1,28 @@
 import { createStore, applyMiddleware, compose } from 'redux';
 import createSagaMiddleware from 'redux-saga';
+// import logger from 'redux-logger';
 import { routerMiddleware } from 'react-router-redux';
 
 import rootReducer from './reducers/index';
-import rootSaga from './Sagas/asyncOperations';
+import rootSaga from './Sagas/index';
```
Comment (tomily1) [inline client/store.js:7]: importing index can also be done this way `import rootSaga from './Sagas/`

**Review by tomily1 — COMMENTED** [review]
Nice work!, Neat code, but you might want to split your PRs into chunks of features next time. because react testing does not explain the sole purpose of this PR, it does not cover react testing alone

File: client/Sagas/asyncOperations.js (hunk 3 above)
Comment (d3mola) [reply client/Sagas/asyncOperations.js:383]: Thanks. I'll remove unused code from my codebase.

File: .babelrc (hunk 1 above)
Comment (d3mola) [reply .babelrc:7]: Noted! Thanks.

**Pull request closed without merging**